qemu/tests/balloon_sc_interrogate.py

```diff
@@ -0,0 +1,74 @@
+import re
+import logging
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import error_context
+from qemu.tests.balloon_check import BallooningTestWin
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Balloon service should be in running status after sc interrogate it.
+    1) boot a guest with balloon device.
+    2) check balloon driver installation status.
+    3) enable and check driver verifier in guest.
+    4) install and start balloon service in guest.
+    5) send INTERROGATE signal to balloon service.
+    6) check balloon service status again.
+    """
+
+    def get_disk_vol(session):
+        """
+        Get virtio-win disk volume letter for windows guest.
+
+        :param session: VM session.
+        """
+        key = "VolumeName like 'virtio-win%'"
+        try:
+            return utils_misc.get_win_disk_vol(session,
+                                               condition=key)
+        except Exception:
+            test.error("Could not get virtio-win disk vol!")
+
+    def interrogate_balloon_service(session):
+        """
+        Sending INTERROGATE to balloon service.
+
+        :param session: VM session.
+        """
+        logging.debug("Send INTERROGATE to balloon service")
+        sc_interrogate_cmd = params["sc_interrogate_cmd"]
+        status, output = session.cmd_status_output(sc_interrogate_cmd)
+        if status:
+            test.fail(output)
+
```

Comment:
I think it should be "test.error" is better, since it is not the fail point in this case.  And you only use "logging.debug" , so it means this step is not so import to lead fail.

Reply:
agree with @kimi1978wy  `test.error`